src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Withdraw.tsx

```diff
@@ -416,22 +416,29 @@ export const Withdraw: React.FC<WithdrawProps> = ({ accountId, onNext }) => {
         })
       }
 
+      if (maybeWithdrawGasEstimateCryptoBaseUnit?.isErr()) {
+        return translate('common.insufficientAmountForGas', { assetSymbol: feeAsset.symbol })
+      }
+
       if (valueCryptoPrecision.isEqualTo(0)) return ''
       return hasValidBalance || 'common.insufficientFunds'
     },
     [
+      opportunityData?.stakedAmountCryptoBaseUnit,
```

Comment:
is this necessary if we are checking reacting on `maybeWithdrawGasEstimateCryptoBaseUnit` which already checks for this?

Reply:
Yeah it's not anymore 🕵🏽 ! Removed in 81fefb102893bd120cdb6d6b9d58e9cea2f7377d